### DISCUSSION TITLE
RBS definitions

### DISCUSSION THREAD

**Discussion by cperezabo** [mongodb/mongo-ruby-driver#2817] [Ideas]
Hi, how about adding RBS definitions? They are needed for tools like Steep

**Comment by alexbevi** [comment]
Hey @cperezabo and thanks for reporting this. We previously had a ticket opened for investigating adding Sorbet support (RUBY-1801), but we have now created RUBY-3360 to look at this more holistically.

I'm going to close this topic as any further discussion would take place there.
